### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "Cancelled"

### DISCUSSION THREAD

**Discussion by blizaga** [community/community#11725] [Copilot Conversations] [answered]
Please help me to solve this problem, Thanks

![image](https://user-images.githubusercontent.com/54091450/154734703-46fd0426-73cb-4488-a2f6-d0e3885f0677.png)

**Comment by iamzaidsoomro** [comment]
It also happened with me at the start, it was not authenticating. Then I connected my Microsoft account and it started working. Give it a try, it may work.

**Reply by blizaga** [reply]
Thanks bro

**Reply by blizaga** [reply]
it is work for me

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by blizaga** [reply]
Thank you

**Reply by Marychoffel22** [reply]
Assign
